encryption/encryption.cc

```diff
@@ -457,7 +461,12 @@ Status KeyManagedEncryptedEnv::RenameFile(const std::string& src_fname,
                                           const std::string& dst_fname) {
   if (ShouldSkipEncryption(dst_fname)) {
     assert(ShouldSkipEncryption(src_fname));
-    return target()->RenameFile(src_fname, dst_fname);
+    Status s = target()->RenameFile(src_fname, dst_fname);
+    // Replacing with plaintext requires deleting the info in the key manager.
+    Status delete_status __attribute__((__unused__)) =
```

Comment:
Let's handle the error by hacking a bit. In `KeyManagedEncryptedEnv::NewSequentialFile`, check if the file being read is the `CURRET` file. If so, try to read it if it is encrypted, and if the file doesn't start with "MANIFEST-" and end with "\n", reopen it as plaintext file.